DigitalLearningSolutions.Data/Services/CourseService.cs

```diff
@@ -134,7 +136,8 @@ public IEnumerable<CourseStatistics> GetCentreSpecificCourseStatistics(int centr
         {

```

Comment:
We could simplify this method a bit, by calling VerifyAdminUserCanViewCourse (which is a weaker condition) then checking the rest of the conditions (I think that's just checking the course isn't an all-centres course, but I haven't made sure of that). That would be worth doing I think.